mitiq/interface/mitiq_pennylane/tests/test_conversions_pennylane.py

```diff
@@ -137,22 +135,11 @@ def test_integration():
         np.random.seed(1967)
         for gates in gates_per_layers:
             for gate in gates:
-                params = list(np.pi * np.random.rand(gate.num_params))
-                rnd_wires = np.random.choice(
-                    range(n_wires), size=gate.num_wires, replace=False
-                )
-                gate(
-                    *params,
-                    wires=[
-                        int(w) for w in rnd_wires
-                    ],  # make sure we do not address wires as 0-d arrays
-                )
```

Comment:
I guess you had to remove this for some tricky reason.
If not possible to re-introduce:
- Can we restore the random rotation parameters only?
- Can we restore the random wires only?

If not, it's fine. This is not a blocker for this PR.

Reply:
The reason for this change is because `gate.num_params` is now dynamic. Since this test is copied from pennylane, the following PR recommends re-writing the test this way. See https://github.com/PennyLaneAI/pennylane/pull/1898/files#r750946650.

Reply:
Ah, I see, very nice finding!